### ISSUE TITLE
"Failure to establish SSH session" after upgrading Windows 10 to version 1909 using WINCNG crypto backend 

### ISSUE THREAD

**Issue by bweber-kx** [libssh2/libssh2#456]
libssh2 was successfully establishing connections when run on Windows 10 1809, however after upgrading Windows to 1909 and recompiling; SSH sessions can no longer be established using the WinCNG backend.

**To Reproduce**

Compile the examples, with WinCNG as the crypto backend and run example-ssh2.exe with the IP of github
```
Pinging github.com [192.30.253.112] with 32 bytes of data:
Reply from 192.30.253.112: bytes=32 time=27ms TTL=49
Reply from 192.30.253.112: bytes=32 time=27ms TTL=49
Reply from 192.30.253.112: bytes=32 time=26ms TTL=49

Ping statistics for 192.30.253.112:
    Packets: Sent = 3, Received = 3, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 26ms, Maximum = 27ms, Average = 26ms
Control-C
^C
c:\libssh2-1.9.0\example\Release>example-ssh2.exe 192.30.253.112
Failure establishing SSH session
```

**Expected behavior**
should be able to run the above example; expect SSH session to establish, but authentication to fail

**Version:**
 - OS: Windows 10 Pro Version 1909 Os Build 18363.657
 - libssh2 version 1.9.0

**CMAKE Flags**

I had run with the following flags
```
cmake -G "Visual Studio 16 2019" -DCRYPTO_BACKEND=WinCNG -DBUILD_SHARED_LIBS=ON -DBUILD_TESTING=OFF -DBUILD_EXAMPLES=ON -DCMAKE_INSTALL_PREFIX=./build
```

**Comment by mback2k** [comment]
This is probably related to #397.

**Comment by willco007** [comment]
@bweber-kx please try PR #397 and report back.

**Comment by jonathanturcotte** [comment]
I'm working with @bweber-kx on this, using the version in #397 fixes the issue for us. Only thing of note is that we ran into a missing dependency on ntdll.lib when building, but I snuck it into the CMakeLists.txt and it built fine after that.

**Comment by jonathanturcotte** [comment]
Any plans for a release with @mback2k's fix? It worked well for us, but we'd like to take an official release instead of master.

**Comment by willco007** [comment]
Fixed with #397, release is pending.
